### ISSUE TITLE
Missing client IDs in tapiriik/local_settings.py

### ISSUE THREAD

**Issue by gosdenj** [cpfair/tapiriik#89]
Hello,

I'm attempting to run tapiriik locally, but I've hit a brick wall, after generating the tapiriik/local_settings.py file, all client IDs are "null", see below:

RUNKEEPER_CLIENT_ID="####"
RUNKEEPER_CLIENT_SECRET="####"

DROPBOX_FULL_APP_KEY = "####"
DROPBOX_FULL_APP_SECRET = "####"

DROPBOX_APP_KEY = "####"
DROPBOX_APP_SECRET = "####"

STRAVA_CLIENT_SECRET = "####"
STRAVA_CLIENT_ID = "####"

STRAVA_RATE_LIMITS = []

SPORTTRACKS_CLIENT_ID = "####"
SPORTTRACKS_CLIENT_SECRET = "####"

ENDOMONDO_CLIENT_KEY = "####"
ENDOMONDO_CLIENT_SECRET = "####"

NIKEPLUS_CLIENT_NAME = "####"
NIKEPLUS_CLIENT_ID = "####"
NIKEPLUS_CLIENT_SECRET = "####"

RWGPS_APIKEY = "####"

When I attempt to connect with RunKeeper, the following URL results in an error 400:

https://runkeeper.com/apps/authorize?client_id=####&response_type=code&redirect_uri=http://localhost:8000/auth/return/runkeeper

As you can see the client_id is not set, based on the contents of local_settings.py.

Please advise how I can get this to work.

Kind regards,
Jeff

**Comment by cpfair** [comment]
Yes, you need to fill those values in with your own API keys obtained from
each site, as described in the setup instructions. RunKeeper, Strava and
Dropbox make these freely available, for the others you'll need to contact
the sites directly.

**Comment by jimmykane** [comment]
Coming back to this how do you get the tokens from the Endomondo API?

**Comment by cpfair** [comment]
You'll have to contact them directly - that's what I did back when I first got set up with their API, and I don't think they've opened up registration any further since.

**Comment by jimmykane** [comment]
Ok thanks. Too pitty if they are closed.
